packages/blast-cmd/node/node.js

```diff
@@ -1,17 +1,35 @@
-const { executeCompose } = require('../../blast-utilities/run-docker-commands')
+const { executeCompose, executeAttach } = require('../../blast-utilities/run-docker-commands')
 const {
   getNodeStatus,
   checkNodeOnline,
   checkNodeOffline
 } = require('../../blast-utilities/get-node-status')
+const { getAdditionalAccounts } = require('../../blast-utilities/config-utils')
+const { handleAdditionalAccountCreation } = require('../../blast-utilities/keypair')
+const { delay } = require('../../blast-utilities/delay')
 
 const startNodeCmd = async function(argv) {
   await checkNodeOffline()
-  if (argv.daemon) {
-    executeCompose('up --build -d')
-    return
+
+  executeCompose('up --build -d')
+
+  while (true) {
+    const nodeStatus = await getNodeStatus()
+    if (nodeStatus.isConnected) {
+      break
+    }
+    await delay(2)
+  }
+  await delay(4)
```

Comment:
if you think it will be helpful, you can put a small comment about that delay because it feels like some workaround